cmd/starbridge/model/transaction.go

```diff
@@ -15,6 +36,7 @@ type Transaction struct {
 	To                   string
 	AssetInfo            *AssetInfo
 	Amount               uint64
+	Data                 ContractData // this is where the actual information lives about what needs to be transferred
```

Comment:
save the contract data from the Payment event on our model Transaction directly so we can use when converting it later. This encapsulated our data very well